containers/daap-query-data/tests/unit/mock_test.py

```diff
@@ -0,0 +1,65 @@
+from query_data import handler
+from unittest.mock import Mock
+import pytest
+import boto3
+
+
+def test_query_athena(athena_client, fake_context):
```

Comment:
it would be good to have a test for an empty result set as well, to make sure the lambda handles both